lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -52,6 +53,7 @@ export class PublicClientApplication extends ClientApplication implements IPubli
      * @param {@link (RedirectRequest:type)}
      */
     async loginRedirect(request?: RedirectRequest): Promise<void> {
+        this.broadcast(BroadcastEvent.LOGIN_START);
```

Comment:
Idea: payload for the "start" events could be an object, that includes a field denoting the interaction type (silent, redirect, popup)